extension/content_script.js

```diff
@@ -557,5 +561,23 @@ function toggleExtensionVisibility() {
 	}
 }
 
-oneTimeSetup();
+function setupKeyListenerForOnOff() {
+	jdoc.on("keydown", function(evt) {
+		if (!document.hasFocus()) {
+		  return true;
+		}
+		// Make sure the user isn't trying to type anything
+		// If there are exceptions to this it should hopefully come up during testing
+		// There probably will be exceptions, so the key is WHAT are the exceptions
```

Comment:
hm can u leave TODO: google in stack overflow?
these things usually have a nice jquery / someone already cover all the edgecases on stackoverflow

or if you already have a stackoverflow link pls link it here